lib/kitchen/element_enumerator_base.rb

```diff
@@ -51,6 +51,17 @@ def pages(css_or_xpath=nil)
       chain_to(PageElementEnumerator, css_or_xpath: css_or_xpath)
     end
 
+    # Returns an enumerator that iterates through pages that arent the introduction page within the scope of this enumerator
+    #
+    # @param css_or_xpath [String] additional selectors to further narrow the element iterated over;
+    #   a "$" in this argument will be replaced with the default selector for the element being
+    #   iterated over.
+    #
+    def non_introduction_pages
+      block_error_if(block_given?)
+      chain_to(PageElementEnumerator, css_or_xpath: '$:not(.introduction)')
```

Comment:
this probably needs to be updated for the `search_query` way in `main`.  Means you'd have `def non_introduction_pages(only: nil, except: nil)` too